framework/src/aifw/readme.md

```diff
@@ -0,0 +1,72 @@
+# **AI Framework Guide**
+## **1. AI Framework Guideline**
+- AI Framework supports single thread architecture.
+- Public interfaces are placed in directory framework/include/aifw.
+- It has dependency on C++ library.
+- Core functionalities are:
+  1. AI Model management: Loading of model, input, output and clearing memory
+  2. Data buffer for application to perform pre-processing and ensemble operations
+  3. Abstraction to core AI library such as Tensor Flow
+  4. File based OTN support
+
+## **2. Model json**
+Model json file contains attributes of a model. These are parsed by AI Framework and passed to application for usage.
+
+- crc32: CRC code to verify OTN download
+- version: Format "yymmdd", Version of model deployed on device. This is required to check model update available.
+- modelfile: Path of model file. Path should be /mnt/AI/<prd>/<filename>
+- features: List of values to parse from raw data. Features array can be empty or NULL also.
+- inferenceinterval: Unit milliseconds. Time interval after which inference operation to be performed. When interval value is 0, AI Framework service is not started. Application will push data when received from source. 
+- modelcode: int 32 bit value. Unique identifier for a set of models required to perform a function.
+- maxrowsdatabuffer: Row capacity of data buffer. This is circular buffer.
+- rawdatacount: Count of float values which will be available for pre-processing. Each row of buffer will contain these count of values.
+- windowsize: Number of rows required for inference operation
+- invokeinputcount: Number of float values for input to model 
+- invokeoutputcount: Number of float values returned from model
+- postProcessResultCount: Number of values as output of post process operation
+- inferenceResultCount: Count of primitive data values sent to application after inference operation
+- preprocessing: Contains list of values for mean and standard deviation. These are required in pre process operation
+
+Sample JSON
+{
+    "crc32": 15000,
+    "version": "yymmdd",
+    "modelfile": "/mnt/AI/<prd>/<filename>",
+    "features": [],
+    "inferenceinterval": 2000,
+    "modelcode": 16843010,
+    "maxrowsdatabuffer": 1,
+    "rawdatacount":6,
+    "windowsize": 1,
+    "invokeinputcount": 6,
+    "invokeoutputcount": 17,
+    "postProcessResultCount": 1,
+    "inferenceResultCount": 1,
+    "preprocessing": {
+        "mean": [],
+        "std": []
+    }
+}
```

Comment:
When you check this in the md file, all are written in a single line.
1. add them using \```
2. or add two spaces at every line

Reply:
Added ```. Thanks for the suggestion.